.github/workflows/test-e2e-gpu.yaml

```diff
@@ -0,0 +1,87 @@
+name: GPU E2E Test
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
@jaiakash Please can you open an issue to track that we should refactor this action with pull_request_target and get HF_TOKEN from GitHub secrets ?